common/common-rest/src/main/java/org/apache/servicecomb/common/rest/codec/produce/ProduceProcessorManager.java

```diff
@@ -17,24 +17,43 @@
 
 package org.apache.servicecomb.common.rest.codec.produce;
 
+import java.util.ServiceLoader;
+
 import javax.ws.rs.core.MediaType;
 
 import org.apache.servicecomb.foundation.common.RegisterManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class ProduceProcessorManager extends RegisterManager<String, ProduceProcessor> {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ProduceProcessorManager.class);
+
+  private static final ServiceLoader<ProduceProcessor> produceProcessor = ServiceLoader.load(ProduceProcessor.class);
```

Comment:
org.apache.servicecomb.foundation.common.utils.SPIServiceUtils

Reply:
add getOrder in  ProduceProcessor 
so that customers have chance to override our logic.

Reply:
done, there is still one question: what number should json and text/plain return when getOrder, i use 10 and 11 now

Reply:
done